ecl/eclagent/eclagent.cpp

```diff
@@ -2502,6 +2502,8 @@ bool EclAgent::checkPersistUptoDate(IRuntimeWorkflowItem & item, const char * lo
         errText.appendf("Rebuilding PERSIST('%s'): Saved CRC isn't present", logicalName);
```

Comment:
So I think it should be
```
 if (!isResult(lfn, (unsigned)-2))
         errText.appendf("Building PERSIST('%s'): It hasn't been calculated before", logicalName);
 else if (isFile && !fileExists(logicalName))
         errText.appendf("Rebuilding PERSIST('%s'): Persistent file does not exist", logicalName);
 else if (!item.queryPersistRefresh())
        return true;
 else if (!isResult(crcName, (unsigned)-2))
         errText.appendf("Rebuilding PERSIST('%s'): Saved CRC isn't present", logicalName);
 else
```
However, I'm really not sure of all the implications of ```isResult(crcName, (unsigned)-2) == false```?